Makefile

```diff
@@ -115,8 +115,8 @@ fmt: headers ## Format the entire code base(s)
 
 .PHONY: clean
 clean: clean-certs
-	cd aurae && make clean
 	cd auraed && make clean
+	cd auraescript && make clean
 	@rm -rvf target/*
```

Comment:
this is ok, but i wonder if we should _also_ call `cargo clean` as we do in the sub-folder Makefiles.

Reply:
What does the nested makefile do? Doesn't that itself call cargo clean?

Reply:
Hi everyone 🙂. Yeah they call `cargo clean`.  This got me thinking if instead we should replace the clean target with a `cargo clean` (which also does an `rm -rf /target`)   while at the same time leverage the existing Makefiles to clean the packages (auraed and auraescript) individually if needed.

Reply:
Sure however you want to do it. I will merge now.